frontend/src/components/Header.js

```diff
@@ -10,10 +10,10 @@ export default function Header({title}){
     return(
         <Wrapper>
             {user && <Avatar src={user.avatarUrl} alt=""/>}
-            {!user && <div></div>}
-            <h1>{title}</h1>
+            {!user && <div/>}
+            <h3>{title}</h3>
             {user && <Button onClick={logout}>Logout</Button>}
-            {!user && <div></div>}
+            {!user && <div/>}
```

Comment:
I don't get why you are conditionally closing the divs, looks a bit weird.
When I don't have a user the markup will look like this
```html
<header>
  </div>
  <h3>some title</h3>
  </div>
</header>
```
This is broken, please check again.